### DISCUSSION TITLE
Palette filter Vs RGB key

### DISCUSSION THREAD

**Discussion by F-Burning** [opentoonz/opentoonz#6553] [Ideas]
Hi

I was wondering if in the future a function can be added to the palette filter so that it works the same as RGB Key leaving only the line visible

Here are the differences:

Palette filter

![1](https://user-images.githubusercontent.com/74098734/157125235-00cf67b4-d64a-410e-9765-9dc915e32de3.jpg)

RGB Key

![2](https://user-images.githubusercontent.com/74098734/157125246-de3ab20e-d9fe-4fa2-95e5-f6055e2e2cca.jpg)

I would like to be able to work with the second option, but in the filter palette, because in this one all the lines of all the levels are visible.

**Comment by RodneyBaker** [comment]
As this report has shifted from a question to a feature request it will be good to update the title and information in this report.
Alternatively, this report could be closed and a new report opened detailing any requests.

Edit:  Changed the label to feature request but am not sure what is ideal for the title.
